app/dashboard/models.py

```diff
@@ -260,6 +260,7 @@ class Bounty(SuperModel):
     project_type = models.CharField(max_length=50, choices=PROJECT_TYPES, default='traditional')
     permission_type = models.CharField(max_length=50, choices=PERMISSION_TYPES, default='permissionless')
     snooze_warnings_for_days = models.IntegerField(default=0)
+    auto_approve_workers = models.BooleanField(default=True)
```

Comment:
Can't we simply reuse the `Bounty.admin_override_suspend_auto_approval` field for this?  If we're worried about the naming convention, rename and migrate the field?
Then you could simply update the JS to display this button if staff OR funder and display `Suspend` or `Enable` based on the current state?

Reply:
We can. I've added new field because this two have slightly different meaning, and I was not sure I can merge them. Also, we was planning to add check to the bounty details page so funder can turn it off/on at any time, and in that case, if we merge those two fields, admin will not be able to override it in a way so that funder will not be able to flip it back. I was not sure if this is acceptable.

Reply:
@mbeacom your thoughts on that one?

Reply:
@mbeacom I've remove extra field, but I don't know how to do data migration

Reply:
@darkdarkdragon - I see your comment about not knowing how to do data migration - is that blocking you right now, or is it a task that needs to be handed off?

Reply:
@frankchen07 either way. If somebody point me to relevant example how to do it, I'll do it, but if someone else can do it faster that's fine too

Reply:
@darkdarkdragon `make migrations` will generate the automatic migrations.  `make migrate` migrates your local DB with the new migration changes.

Reply:
@darkdarkdragon What I was recommending was to delete **your** new migrations and field and simply use the original field for this.  If you are unsure of how to migrate a field to a new field name (not entirely required at the moment), please just delete your migrations in this branch, remove your new field, leave the original field and use that.

Reply:
Simply switch back to using `admin_override_suspend_auto_approval` instead of `auto_approve_workers` for the time being.